flink-libraries/flink-table/src/main/scala/org/apache/flink/table/runtime/MapJoinLeftRunner.scala

```diff
@@ -31,7 +32,19 @@ class MapJoinLeftRunner[IN1, IN2, OUT](
   override def flatMap(multiInput: IN1, out: Collector[OUT]): Unit = {
     broadcastSet match {
       case Some(singleInput) => function.join(multiInput, singleInput, out)
-      case None =>
+      case None => {
```

Comment:
Really, I cant imagine situation when there we will got null. Please see in /flink/flink-libraries/flink-table/src/main/scala/org/apache/flink/table/runtime/MapSideJoinRunner.scala who is parent of these runners. 
org/apache/flink/table/runtime/MapSideJoinRunner.scala:48
`    broadcastSet = retrieveBroadcastSet
  }

  private def retrieveBroadcastSet: Option[SINGLE_IN] = {
    val broadcastSet = getRuntimeContext.getBroadcastVariable(broadcastSetName)
    if (!broadcastSet.isEmpty) {
      Option(broadcastSet.get(0))
    } else {
      Option.empty
    }
  }`